quic/s2n-quic-platform/src/syscall.rs

```diff
@@ -40,15 +32,26 @@ pub fn bind_udp<A: std::net::ToSocketAddrs>(addr: A, reuse_port: bool) -> io::Re
             let socket = Socket::new(domain, socket_type, protocol)?;
             socket.set_nonblocking(true)?;
         }
-    };
-
-    // allow ipv4 to also connect
-    if addr.is_ipv6() {
-        socket.set_only_v6(false)?;
     }
 
+    // allow ipv4 to also connect - ignore the error if it fails
+    let _ = socket.set_only_v6(false);
```

Comment:
what was the reason for this change?

Reply:
It didn't really make sense to fail on this, similar to the other platform-specific features like TOS, MTU Discovery, etc.